### ISSUE TITLE
Not Receiving notifications on some processors

### ISSUE THREAD

**Issue by Lowvoltage-ca** [culvr/ProwlSimplSharp#3]
i keep getting this error on mutlitple projects, yet I have one project that is still running with no issues.

Exception 'Data Abort' (4): Thread-Id=0c4a003e(pth=8c136324), Proc-Id=0735000a(pprc=88771b9c) 'splusmanagerapp.exe', VM-active=0735000a(pprc=88771b9c) 'splusmanagerapp.exe'
PC=0028b6f8(???+0x0028b6f8) RA=00000000(???+0x00000000) SP=05fafb80, BVA=00000000
Error: splusmanagerapp.exe [App 1] # 15:31:30  3-04-2016  # Module S-19.9.1 : Prowl(Prowl Notifications) at line 121: Unhandled Exception: System.NullReferenceException: NullReferenceException
   at ProwlSimplSharp.ProwlClient.ProwlDispatch(ProwlRequest request)
   at ProwlSimplSharp.ProwlClient.Send(SError: splusmanagerapp.exe [App 1] # 15:31:30  3-04-2016  # tring app, UInt16 priority, String url, String subject, String message)
   at UserModule_PROWL.UserModuleClass_PROWL.SEND_OnPush_2(Object **EventInfo**)
   at Amib.Threading.Internal.WorkItem.e()
   at Amib.Threading.Internal.WorkItem.ExecuError: splusmanagerapp.exe [App 1] # 15:31:30  3-04-2016  # te()
   at Amib.Threading.SmartThreadPool.n(WorkItem A_0)
   at Amib.Threading.SmartThreadPool.e()

**Comment by culvr** [comment]
Are you sending a string into all of the inputs? app, url, subject message?

**Comment by Lowvoltage-ca** [comment]
Yes, they have all been working for months, I still have 1 site working currently. And I haven't touched firmware on any of ‎ them or programming. They just stopped working. I can send test messages from prowl website to my phones with no problems as well. Been stumped for over a week with this one.                                                                                                                                                                                                                                                                                                                                         Regards,‎Allan Silvaallan@lowvoltage.ca416-779-8484This message may contain information that is confidential orprivileged. Any use of this information by anyone other than theintended recipient is prohibited. If you have received this message inerror, please immediately reply to the sender and delete thisinformation from your computer. Thank you.                                                                                                                                                                                                                From: Andrew CulverSent: Saturday, March 5, 2016 11:07 AMTo: culvr/ProwlSimplSharpReply To: culvr/ProwlSimplSharpCc: Lowvoltage-caSubject: Re: [ProwlSimplSharp] Not Receiving notifications on some processors (#3)Are you sending a string into all of the inputs? app, url, subject message?

—Reply to this email directly or view it on GitHub.

**Comment by prasannakulkarni1888** [comment]
I too facing same issue, Before it was working perfectly fine.

**Comment by Lowvoltage-ca** [comment]
Hey Andrew,

I spoke to Crestron Last Week and here is what I got from them.

"I was able to replicate the issue. It seems that the MC3 doesn’t negotiate a secure connection with the PROWL API. There appears to be a discrepancy with the number of root certificates between the MC3 and its brethren 3-series. I’ve asked Engineering for a review of this, but it may take a couple of days
                In the meantime, you could probably re-write a module to not use a secure connection…   "

Is there anyway to re-write the module to not use ssl? I'm looking for a temp solution until Crestron gets it fixed on their end.
